products/compute/ansible.yaml

```diff
@@ -24,6 +24,82 @@ manifest: !ruby/object:Provider::Ansible::Manifest
   version_added: '2.6'
   author: Google Inc. (@googlecloudplatform)
 # This is where custom code would be defined eventually.
+datasources: !ruby/object:Provider::ResourceOverrides
```

Comment:
Sorry to keep hassling you here, but it seems like off-by-default would be the right approach.  Most resources don't need datasources.

Reply:
In the Ansible world, we're looking to build datasources for everything. I'm inclined to keep this as-is because my intent is for most of these lines to go away (very, very soon).

Reply:
Great, in the Terraform world we agree also!